smurfs/src/components/SmurfForm.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import { fetchSmurfs, createSmurf } from '../actions';
+
+const SmurfForm = props => {
+    console.log('these are the props from the Smurf Form', props)
+
+    const [smurf, setSmurf] = useState({});
+    console.log('this is my smurf form component state', smurf)
+
+    useEffect(() => {
+        props.fetchSmurfs();
+    }, [])
+
+    if (props.isFetching) {
+        console.log('this is my loading props',props.isFetching)
+        return <h2>Loading Your Smurfs...</h2>;
+      }
+    const handleChange = e => {
+        setSmurf({...smurf, [e.target.name]: e.target.value})
+    }
+
+
+    const handleSubmit = e => {
+        e.preventDefault();
+        if (smurf.name.trim() && smurf.age.trim() && smurf.height.trim()) {
```

Comment:
I like your use of .trim() but could you add any error handling around users entering incorrectly?